obspy/clients/fdsn/routers/__init__.py

```diff
@@ -109,7 +109,7 @@
 ...                             starttime='2015-05-24T12:00:00')
 
   or, using obspy.core.UTCDateTime ...
->>> t1 = UTCDateTime(2015,05,24,12,0,0)
+>>> t1 = UTCDateTime(2015,5,24,12,0,0)
```

Comment:
@nick-iris please do also adhere to PEP8 in docstrings/doctests, e.g. here add spaces after commas.